docker-compose.yml

```diff
@@ -53,7 +53,7 @@ services:
     networks:
       - altinncore_network
     volumes:
-      - "C:/AltinnCore/Repos:/AltinnCore/Repos"
+      - "/Users/jezpoz/AltinnCore/Repos:/AltinnCore/Repos"
```

Comment:
Hardcoded

Reply:
It was already hardcoded 😅. Just that this works for OSX. We should use relative path, and ignore the files in `.gitignore`.

Reply:
Will change back, since 99% of our developers use windows.